src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  const { name, email, body } = comment;
+
+  return (
+    <div className="CommentInfo comment">
+      <div className="content">
+        <a href={`mailto:${email}`} className="CommentInfo__name author">{name}</a>
+        <div className="metadata">
+          <a href={`mailto:${email}`} className="CommentInfo__email date">{email}</a>
+        </div>
+        <p className="CommentInfo__body">{body}</p>
+      </div>
+    </div>
+  );
```

Comment:
![Screenshot_44](https://user-images.githubusercontent.com/110488158/202421688-dcc54cb6-c230-4944-af2b-2609303fbe1d.png)
The original structure was quite different from your option